packages/react-ui-components/src/Tabs/tabs.js

```diff
@@ -73,13 +73,20 @@ export default class Tabs extends PureComponent {
         );
     }
 
+    getActiveTab() {
```

Comment:
I'd call this "activeTabIndex"; and also change the state property to "activeTabIndex" - is more clear, wdyt?

Reply:
The problem with renaming state `activeTab` is that it has related prop `activeTab`, which is a part of component's API, and since we publish this package to npm and already version it, that would require a version bump... Frankly I'd just ignore it, but if you insist I can take care.

Reply:
OK then we keep it as is. Fine ;)